### PR TITLE
monarch 0.6.70 (new cask)

### PR DESCRIPTION
**Pull request by rmdashrfv**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/m/monarch.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  arch arm: "arm64", intel: "x64"
+
+  version "0.6.70"
+  sha256 arm:   "e4b557c50f5ffa37ab0d2dc18dca43935bc6a5a6ce919531b2dd656faab79f14",
+         intel: "7db2c4940c483e3c96df64f0bb37ae6fcbbda90d45cb61075897031ac1765a8c"
+
+  url "https://monarchlauncher.s3.amazonaws.com/#{arch}/monarch-#{version}.dmg",
+      verified: "monarchlauncher.s3.amazonaws.com/"
+  name "Monarch"
+  desc "Spotlight Search with Superpowers"
```
Comment (khipp) [inline Casks/m/monarch.rb:11]:
```suggestion
  desc "Spotlight Search"
```

See [documentation](https://docs.brew.sh/Cask-Cookbook#stanza-desc).

File: Casks/m/monarch.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  homepage "https://monarchlauncher.com/"
+
+  livecheck do
+    url "https://monarchlauncher.s3.amazonaws.com/#{arch}/latest-mac.yml"
+    strategy :yaml do |yaml|
+      yaml["version"]
+    end
```
Comment (khipp) [inline Casks/m/monarch.rb:18]:
```suggestion
    strategy :electron_builder
```

File: Casks/m/monarch.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  end
+
+  depends_on macos: ">= :monterey"
```
Comment (khipp) [inline Casks/m/monarch.rb:21]:
```suggestion
  depends_on macos: ">= :catalina"
```

**Review by khipp — COMMENTED** [review]
Thank you for your submission, @rmdashrfv! I left a few comments.

File: Casks/m/monarch.rb (hunk 3 above)
Comment (rmdashrfv) [reply Casks/m/monarch.rb:19]: I've not developed the app or tested it for Catalina, nor have I announced support for Catalina. Is this an issue?

Comment (bevanjkay) [reply Casks/m/monarch.rb:19]: The audit is showing that the app's info.plist file suggests Catalina as the minimum required macOS version.

Comment (rmdashrfv) [reply Casks/m/monarch.rb:19]: That's an oversight by me then. My mistake. I will correct this and resubmit. Sorry for the confusion.

Comment (rmdashrfv) [reply Casks/m/monarch.rb:19]: @khipp @bevanjkay Info.plist has been updated appropriately. Sorry for the confusion here

**Review by razvanazamfirei — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
Hi, @rmdashrfv! This looks great! 👍 Thanks so much for your first contribution! Without contributions like yours, it'd be impossible to keep our project going.

**Comment by rmdashrfv** [comment]
@razvanazamfirei Thanks for getting this across the line! Really appreciate you guys! I donated! 🍺

**Pull request merged**